Tensile/Source/client/CMakeLists.txt

```diff
@@ -47,6 +47,9 @@ if(NOT WIN32)
     )
 endif()
 
+add_definitions(-U_GLIBCXX_ASSERTIONS)
+add_definitions(-U_LIBCPP_ENABLE_ASSERTIONS)
+
```

Comment:
I don't think this is making any difference.

Reply:
Removed in [Address comments](https://github.com/ROCm/Tensile/pull/1945/commits/ca90a4d3f085efd122989e729f2e0d803335a7fc)